services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/Persistence.kt

```diff
@@ -114,7 +115,13 @@ class HankeEntity (
         // can be a performance problem if there is a need to do bulk inserts.
         // Using SEQUENCE would allow getting multiple ids more efficiently.
         @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
-        var id: Int? = null
+
```

Comment:
No extra newline between annotations and related field

Reply:
Yes. I guess it came from merge.